Missing -I flag for libxfce4panel/xfce-panel-plugin.h: No such file or directory

--- src/Makefile.am
+++ src/Makefile.am
@@ -27,7 +27,8 @@
 	-DPACKAGE_LOCALE_DIR=\"$(localedir)\" \
 	@LIBSENSORS_CFLAGS@	\
 	@LIBXFCE4UTIL_CFLAGS@ \
-	@LIBXFCEGUI4_CFLAGS@
+	@LIBXFCEGUI4_CFLAGS@ \
+	@LIBXFCE4PANEL_CFLAGS@
 
 # @GTK_CFLAGS@ 	\
 # @GLIB_CFLAGS@ \
